### PR TITLE
V247 rc2 prep

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -158,6 +158,17 @@ CHANGES WITH 247 in spe:
           mounting additional disk images into the file system tree accessible
           to the service.
 
+        * Timer units gained a new FixedRandomDelay= boolean setting. If
+          enabled the random delay configured with RandomizedDelaySec= is
+          hashed from the unit name, system identity, and execution context, so
+          that always the same offset is used for the same unit on the same
```
Comment (bluca) [inline NEWS:164]: ```that always the same offset is used``` -> ```that the same offset is always used```

**Review by bluca — APPROVED** [review]
minor comment about wording in NEWS, apart from that lgtm

File: NEWS (hunk 1 above)
Comment (keszybz) [reply NEWS:164]: I'd skip those precise details of implementation. The man page uses the same phrasing, and I think it's too much there too. I'll submit a follow-up pr.

**Comment by keszybz** [comment]
travis: `236/796 hwdb-test TIMEOUT 270.02s`. I've seen the same failure before, but I don't understand it. On my machine, it takes a few seconds to do this. Maybe the test VM got stalled, and the test just failed randomly.
samaphore: timed out in setup.
Looks unrelated, let's merge.

**Pull request merged**